masterlist.yaml

```diff
@@ -4862,6 +4862,46 @@ plugins:
         crc: 0x1D27B5D2
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         nav: 1
+  - name: 'ArcheryV2.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/10926/' ]
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Beyond Skyrim - Bruma' ]
+        condition: 'active("BSHeartland.esm") and not active("Archery_Bruma_Patch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Campfire - Complete Camping System' ]
+        condition: 'active("Campfire.esm") and not active("Archery_Campfire_Patch.esp")'
+    clean:
+      # version: 2.5
+      - crc: 0xB2AD660F
+        util: 'SSEEdit v4.0.2'
+      # version: 2.4
+      - crc: 0x0DDC51B6
+        util: 'SSEEdit v4.0.2'
+  - name: 'Archery_Bruma_Patch.esp'
+    req:
+      - 'ArcheryV2.esp'
+    after:
+      - 'BSHeartland.esm'
```

Comment:
An `after` for `ESM` or plugin masters in not required.

Reply:
Removed `after`.